### ISSUE TITLE
Description missing in plugin manager

### ISSUE THREAD

**Issue by zbynek** [jenkinsci/material-theme-plugin#15]
### What feature do you want to see added?

For this plugin to be shown nicely in plugin manager and plugin site, it's necessary to upload the jar file to Artifactory. For the latest build it's missing https://repo.jenkins-ci.org/releases/io/jenkins/plugins/material-theme/0.4.1/material-theme-0.4.1.jar

See also https://issues.jenkins.io/browse/JENKINS-68300

**Comment by offa** [comment]
There's an index.jelly now. :+1:

**Comment by zbynek** [comment]
@offa let's wait until the 0.5 release is out and check if the plugin site gets updated, then close this issue.

**Comment by timbrown5** [comment]
Should be fixed in `0.5.0-rc83.6557e02980de`, please reopen if not.
```lang=sh
$ unzip '/Users/timb/Downloads/material-theme-0.5.0-rc83.6557e02980de.jar'     
Archive:  /Users/timb/Downloads/material-theme-0.5.0-rc83.6557e02980de.jar
...
inflating: index.jelly
...
```

**Comment by zbynek** [comment]
Looks good, thanks!
